tests/framework/ROM/TimeSeries/ARMA/arma_reseed_precursor.xml

```diff
@@ -21,14 +21,14 @@
   </TestInfo>
 
   <RunInfo>
-    <WorkingDir>ARMAreseed</WorkingDir>
+    <WorkingDir>gold/ARMAreseed</WorkingDir>
```

Comment:
why the working dir need to be within the gold the folder?

Reply:
Could you explain it? I do not understand the test structure here.

Reply:
It is a little strange. The `arma_reseed_test.xml` test unpickles an ARMA ROM and checks that the histories it generates are identical. The ARMA ROM and gold files for the `arma_reseed_test.xml` are generated by the `arma_reseed_precursor.xml` test. Since we want all gold files to be relative to the gold directory, I had to make `arma_reseed_precursor.xml` place the files in the gold directory so that the `arma_reseed_test.xml` test could find them correctly.

Reply:
I see, I guess you can also use relative path for "arma_reseed_test.xml" test. The current test setup is a little strange for me. If we keep "arma_reseed_precursor.xml" without changes, and use relative path for "arma_reseed_test.xml"  in "tests", does it look better? I will let you decide which way want.

Reply:
Yeah, I don't like writing gold directory on the fly. I'll change it to use relative paths for `arma_reseed_test.xml`.

Reply:
Just to confirm, the `arma_reseed_test` should use the ARMA generated by the `arma_reseed_precursor` and not the one stored in `gold`, right?

Reply:
@PaulTalbot-INL yes, the `arma_reseed_test` uses the ARMA generated by the `arma_reseed_precursor` test. Before the most recent commit, this ARMA was generated and placed in the `gold` directory. Now it is generated and placed in the `ARMAreseed` directory not the `gold` directory.